lib/matplotlib/tests/test_axes.py

```diff
@@ -8861,3 +8861,15 @@ def test_axhvlinespan_interpolation():
     ax.axhline(1, c="C0", alpha=.5)
     ax.axhspan(.8, .9, fc="C1", alpha=.5)
     ax.axhspan(.6, .7, .8, .9, fc="C2", alpha=.5)
+
+
+@check_figures_equal()
```

Comment:
```suggestion

@needs_usetex
@check_figures_equal()
```

Doesn't look like it is already imported in this file, so also add at the top:

```python
from matplotlib.testing._markers import needs_usetex
```

This will skip the test when latex is not installed (this is what is failing the tests)

Reply:
Ohh cool, I was looking for a fix but then I got caught up in some work. Thanks so much!